src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -573,6 +573,15 @@ public void RefreshAndSearchAsync()
         private void AddToSearchResults(PackageManagerSearchElementViewModel element)
         {
             element.RequestDownload += this.PackageOnExecuted;
+
+            //added dummy values for representation, to be removed
+            if (element.Model.Name == "dronovBIM_v2")
+            {
+                element.Versions[0].Item1.full_dependency_ids.Add(new Dependency() { name = "CPython", _id = "3.0" });
+                element.Versions[1].Item1.full_dependency_ids.Add(new Dependency() { name = "IronPython", _id = "2.0" });
+                element.Versions[2].Item1.full_dependency_ids.Add(new Dependency() { name = "IronPython", _id = "2.0" });
+                element.Versions[2].Item1.full_dependency_ids.Add(new Dependency() { name = "CPython", _id = "3.0" });
```

Comment:
We can add another property such as `version` or `vers` to record version instead of `_id` from PMClient class while implementating.

Reply:
Yes i think `version` would be a better choice.